atomic/f23_spc/Dockerfile

```diff
@@ -2,14 +2,11 @@
 
 FROM fedora:23
 
-RUN dnf -y install openscap-scanner openscap-utils openscap-containers scap-security-guide && dnf clean all
 RUN dnf -y install atomic && dnf clean all
 
-# TODO: Don't clone from scratch, instead add local working copy there
-RUN dnf -y install git python3 python3-devel python3-gobject-base \
-    python3-setuptools dbus python3-dbus && dnf clean all && \
-    git clone https://github.com/OpenSCAP/openscap-daemon && \
-    pushd /openscap-daemon && python3 setup.py install && popd
+RUN dnf -y install openscap-containers scap-security-guide openscap-daemon && dnf clean all
+
+RUN dnf -y install wget && dnf clean all
 
```

Comment:
Is it possible to install all packages in one dnf command? If we have only one RUN instruction, the built image will have less layers.